### DISCUSSION TITLE
Filtering

### DISCUSSION THREAD

**Discussion by duxswhatdux** [marcusolsson/obsidian-projects#848] [General]
Hi, I'm glad I came upon this plugin. I was wondering if OR could be included as a join option on Filters?
I work with a software that has a reporting feature, it has a Boolean of AND / OR, along with grouping of AND with ORs using open and close brackets. At the very least, include OR to filters?
![image](https://github.com/marcusolsson/obsidian-projects/assets/45373146/b51ec5b8-1a0c-4298-8445-7a67f07797b9)
Example:
![image](https://github.com/marcusolsson/obsidian-projects/assets/45373146/b086ea2a-a691-42df-96d3-a59de1bb5ecf)

**Comment by Acylation** [comment]
Yeah this has been requested by many and logged in #279. Thank you so much for sharing such a UI implementation!
